desktop-style.css

```diff
@@ -0,0 +1,252 @@
+body {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  height: 3504.94px;
+  margin: 0;
+}
+
+.mobile-nav { display: none; }
+
+header {
+  height: 636px;
+  background: url("assets/images/Header-llustration-desktop.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: top 0 left 27px;
+  padding: 0;
+}
+
+.wrapper {
+  background:
+    url("assets/images/right-about-me.svg"),
+    url("assets/images/left-about-me.svg"),
+    url("assets/images/illustration-1-contact-form-desktop.svg"),
+    url("assets/images/illustration-2-contact-form-desktop.svg"),
+    url("assets/images/illustratoin-3-contact-form-desktop.svg");
+  background-repeat: no-repeat;
+  background-position:
+    top 1900px right 0,
+    top 2150px left -50px,
+    bottom 72.49px left 0,
+    bottom 156.14px left 274px,
+    bottom 74.83px right 0;
+}
+
+.navigation {
+  display: flex;
+  justify-content: center;
+}
+
+.desktop-nav {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+  align-items: center;
+  color: #344563;
+}
+
+.desktop-nav li { padding: 10px 12px; }
+
+.headline {
+  display: flex;
+  width: 65%;
+  margin: auto;
+  padding: 160px 0;
+  height: 436px;
+}
+
+.headline h1 { height: 114px; }
+
+.headline p {
+  height: 96px;
+  margin: 24px 0 0;
+}
+
+.socials { margin-top: 24px; }
+
+.arrow {
+  display: flex;
+  height: 114px;
+  justify-content: center;
+  align-items: flex-end;
+  margin-top: 24px;
+}
+
+.bar { display: none; }
+
+.recent-works {
+  height: 1084px;
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  padding: 80px 0;
+}
+
+.title,
+.title h2 {
+  display: block;
+  width: 447px;
+  height: 48px;
+  margin: auto;
+}
+
+.recent-works-cards {
+  height: 988px;
+  width: 100%;
+  margin: 52px auto 0;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  row-gap: 32px;
+  column-gap: 24px;
+}
+
+.work-details {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.work-details h3 {
+  padding: 0;
+  margin: 0 !important;
+  height: 57px;
+}
+
+.work-details a {
+  margin: 0;
+}
+
+.language-tags {
+  height: 32px;
+  margin: 0;
+}
+
+.about-me {
```

Comment:
- [ ] The button in this section seems to jump out of place when the width is been adjusted. Kindly adjust so that it remains stable across widths

![image](https://user-images.githubusercontent.com/43995830/154493012-c84f6145-3510-4da8-b80d-630cd5e60553.png)